deeprankcore/dataset.py

```diff
@@ -378,6 +380,29 @@ def _compute_mean_std(self):
         self.means = means
         self.devs = devs
 
+    def _check_inherited_param(
+        self,
+        inherited_param: List[str],
+        dataset_train: Union[GraphDataset, GridDataset],
+    ):
+        """Check if the parameters from validation or testing are same as the training set.
+        
+        Args:
+        inherited_param (List[str]): List of parameters that need to be checked for inheritance.
```

Comment:
```suggestion
        inherited_params (List[str]): List of parameters that need to be checked for inheritance.
```

Please change it also below